src/main/java/org/jabref/gui/groups/GroupDialogView.java

```diff
@@ -90,14 +90,15 @@ public GroupDialogView(DialogService dialogService, BibDatabaseContext currentDa
 
     @FXML
     public void initialize() {
+        System.out.println("Group Dialog View initiated");
```

Comment:
In JabRef, we are using a different kind of infrastructure to log messages:
````
private static final Logger LOGGER = LoggerFactory.getLogger(xyz.class);
[...]
LOGGER.debug("My debug message", ex);
````
But writing a debug message for every dialog, which is opened is not required, so this can be omitted.